packages/rest/src/lib/RequestManager.ts

```diff
@@ -291,7 +288,7 @@ export class RequestManager extends EventEmitter {
 			this.handlers.get(`${hash.value}:${routeId.majorParameter}`) ??
 			this.createHandler(hash.value, routeId.majorParameter);
 
-		// Resolve the request into usable fetch/node-fetch options
+		// Resolve the request into usable fetch
```

Comment:
```suggestion
		// Resolve the request into usable fetch options
```

Reply:
Coincidentally fixed in c6d27d055d82503ecfb4b6282f9cd5166698b98e (again lol)